games/devtest/mods/gltf/init.lua

```diff
@@ -0,0 +1,48 @@
+local function register_entity(name, textures)
+	minetest.register_entity("gltf:" .. name, {
+		initial_properties = {
+			visual = "mesh",
+			mesh = "gltf_" .. name .. ".gltf",
+			textures = textures,
+		},
+	})
+end
+
+-- These do not have texture coordinates; they simple render as black surfaces.
+register_entity("minimal_triangle", {})
+register_entity("triangle_with_vertex_stride", {})
+register_entity("triangle_without_indices", {})
```

Comment:
This triangles are only visible from one direction. That's a cool effect, but I initially thought it was an issue with the implementation. Could a back side be added, preferably a few times larger so that it's easier to notice?

Reply:
I'll turn off backface culling for the triangles. I'd prefer not to mess with these models since that may break the C++ unit tests.